libr/anal/block.c

```diff
@@ -461,6 +462,20 @@ R_API void r_anal_block_add_switch_case(RAnalBlock *block, ut64 switch_addr, ut6
 	r_anal_switch_op_add_case (block->switch_op, case_addr, 0, case_addr);
 }
 
+R_API bool r_anal_block_op_starts_at(RAnalBlock *bb, ut64 addr) {
+	if (!r_anal_block_contains (bb, addr)) {
+		return false;
+	}
+	ut16 off = addr - bb->addr;
```

Comment:
i think you should:
* use ut64
* check if distance is bigger than UT16_MAX
* check if addr > bb->addr

Reply:
>     * check if addr > bb->addr

This is done by `r_anal_block_contains()`
Agree on the other points.